src/modules/deal-investor/dto/create-deal-investor-request.dto.ts

```diff
@@ -0,0 +1,67 @@
+import { ValidatedDateOnlyApiProperty } from '@ukef/decorators/validated-date-only-api-property.decorator';
+import { ValidatedStringApiProperty } from '@ukef/decorators/validated-string-api-property.decorator';
+import { DateString, UkefId } from '@ukef/helpers';
+
+export type CreateDealInvestorRequest = CreateDealInvestorRequestItem[];
+
+export class CreateDealInvestorRequestItem {
+  @ValidatedStringApiProperty({
+    description: 'The unique identifier of the portfolio associated to the business entity. The UKEF portfolio is E1.',
+    example: 'E1',
+    minLength: 0,
+    maxLength: 2,
+    required: false,
+    default: 'E1',
+  })
+  readonly portfolioIdentifier: string;
+
+  @ValidatedStringApiProperty({
+    description: "The unique identifier for the deal within the portfolio. It will be a 10-digit code beginning with either '0020', '0030', or '0040'.",
+    example: '0020900111',
+    length: 10, // TO-DO: check
+    pattern: /^00[2-4]0\d{6}$/,
+  })
+  readonly dealIdentifier: UkefId;
+
+  @ValidatedStringApiProperty({
+    description: 'The lender type code for the investor party of the deal.',
+    example: '500',
+    minLength: 0,
+    maxLength: 3,
+    required: false,
+    default: '500',
+  })
+  readonly lenderType?: string; // TO-DO: should this be numeric?
+
+  @ValidatedDateOnlyApiProperty({
+    description: 'The effective date on the deal record in YYYY-MM-DD format.',
+    example: '2023-03-24',
+  })
+  readonly effectiveDate: DateString;
+
+  @ValidatedDateOnlyApiProperty({
+    description: 'The expiry date on the deal record. The default, null, will set it to 99999999 in the system.',
+    example: '2023-03-24',
+    required: false,
+    default: null,
```

Comment:
It doesn't look to me like the default is actually `null` here, it looks like it is `2050-12-31` instead from the logic we've implemented - and the logic MuleSoft has. What do you think?

Reply:
Agreed. I've also changed the second sentence of the description to be: `'If the value provided is null, it will be replaced by 2050-12-31.'`

Reply:
I found 2050-12-31 in Mulesoft dev, qa, staging configs, but looks like it is missing in Production. So It looks like bug in Mulesoft solution.

I am happy to use 2050-12-31 in NodeJS or null

Reply:
Ah I see, thanks @avaitonis 
@oscar-richardson-softwire I think we actually need to make a change here then:
- change the description to something like `'If the value is not provided or is null then the maximum expiry date will be set.'`
- remove the default (or set it to null?) in the swagger spec
- change `PROPERTIES.DEAL_INVESTOR.DEFAULT.expirationDate` to `null`

@avaitonis I know you said you don't mind if we use `null` or `2050-12-31`, but the ACBS documentation says

- _IsExpirationDateMaximum: When ExpirationDate is null, indicates whether its value is '99999999' in the DB_

which sounds like we should send `null` as the ExpirationDate. Given that it matches production, that sounds like the safest choice. Does that sound alright to you @avaitonis?

Reply:
I've made these changes. @avaitonis as Luke is away, if you are happy please could you resolve this thread so I can get this merged in today

Reply:
LG